ipyleaflet/leaflet.py

```diff
@@ -760,6 +760,61 @@ class VideoOverlay(RasterLayer):
     attribution = Unicode().tag(sync=True, o=True)
 
 
+class ImageServiceLayer(ImageOverlay):
+    """ImageServiceLayer class
+
+    Image Service layer for raster data served through a web service
+
+    Attributes
+    ----------
+    url: string, default ""
+        Url to the image service.
+    format: string, default "png"
+        format of exported image (use `'png'` for layers with transparency)
+    pixelType: string, default "F32"
+        data type of the raster image
+    noData: list, default []
+        pixel value or comma-delimited list of pixel values representing no data
+    noDataInterpretation: string, default ""
+        how to interpret no data values
+    interpolation: string, default ""
+        resampling process for interpolating the pixel values
+    compressionQuality: int, default 100
+        lossy quality for image compression
+    bandIds: string, default ""
+        Order of bands to export for multiple band images
+    time: List, default None
+        time instance or extent for image
+    renderingRule: dict, default {}
+        rules for rendering
+    mosaicRule: dict, default {}
+        rules for mosaicking
+    transparent: boolean, default False
+        If true, the image service will return images with transparency
+    crs: dict, default ipyleaflet.projections.EPSG3857
+        Projection used for this image service.
+    """
+
+    _view_name = Unicode('LeafletImageServiceLayerView').tag(sync=True)
+    _model_name = Unicode('LeafletImageServiceLayerModel').tag(sync=True)
+
+    url = Unicode().tag(sync=True)
+    f = Unicode('image').tag(sync=True, o=True)
+    format = Unicode('png').tag(sync=True, o=True)
+    pixelType = Unicode(default_value="F32", allow_none=True).tag(sync=True, o=True)
```

Comment:
We use snake case for properties exposed to python, so `pixel_type` in this case. Same comment for the other properties in this class.